client/state/selectors/test/get-plugin-upload-error.js

```diff
@@ -0,0 +1,42 @@
+/**
+ * External dependencies
+ */
+import { expect } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { getPluginUploadError } from 'state/selectors';
+
+const siteId = 77203074;
+const error = {
+	error: 'folder_exists',
+	message: 'folder_exists',
+};
+
+describe( 'getPluginUploadError', () => {
+	it( 'should return null by default', () => {
+		const state = deepFreeze( {
+			plugins: {
+				upload: {
+					uploadError: {}
+				}
+			}
+		} );
+		expect( getPluginUploadError( state, siteId ) ).to.be.null;
+	} );
+
+	it( 'should return current value for site', () => {
```

Comment:
Yeah I used to do those, but thinking now that `keyedReducer` makes those tests unnecessary? Unless `keyedReducer` breaks, but that is a different concern.

My general view on tests for actions/reducers/selectors is that we have broken these things down into such small parts that most of the value of the tests are as a part of the development process, so it is a judgement call as to how far to go.

Unless we are talking about something like a complex selector, where unit tests can be useful as documentation and guarding against regressions.